### PR TITLE
Added header file to layout

### PR DESCRIPTION
**Pull request by SachseInTheHub**
replaced techdu logo
updated structure

### CODE REVIEW

File: source/index.html.erb [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
+	<p style="text-align: center; margin-top: 10px; margin-bottom: 10px;">Subscribe to our mailing list for tickets reminders and info about future events</p>
+	<form class="header__input" action="//madebymany.us3.list-manage.com/subscribe?u=bb6ef8c1b569c34510855551f&id=2c5505341e" method="post">
+		<div class="sign-up__form__field">
+      		<input placeholder="Enter email address" name="email" type="text" class="sign-up__form__email">
+    	</div>
+    	<div class="sign-up__form__field">
+      		<button type="submit" class="sign-up__form__submit">Submit</button>
+    	</div>
+    </form>
 	</div>
```
Comment (lukaszsagol) [inline source/index.html.erb:19]: If you only made this part aligned properly, would make it easier to read.

File: source/stylesheets/_header.css.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.header {
+	position: relative;
```
Comment (lukaszsagol) [inline source/stylesheets/_header.css.scss:2]: use 2 spaces plz

**Comment by SachseInTheHub** [comment]
updated

**Comment by lukaszsagol** [comment]
:boom:

**Pull request merged**